packages/@lwc/template-compiler/src/parser/attribute.ts

```diff
@@ -37,14 +38,12 @@ import {
 
 import { isCustomElement } from '../shared/ir';
 
-function isQuotedAttribute(rawAttribute: string) {
-    const [, value] = rawAttribute.split('=');
-    return value && value.startsWith('"') && value.endsWith('"');
+function isQuotedAttribute(rawValue: string): boolean {
+    return rawValue ? rawValue.startsWith('"') && rawValue.endsWith('"') : false;
```

Comment:
`rawValue` is guaranteed to be a `string` here, you don't need the existence check.

```suggestion
    return rawValue.startsWith('"') && rawValue.endsWith('"');
```